ttnn/cpp/ttnn/tensor/tensor_impl.cpp

```diff
@@ -809,6 +816,197 @@ Tensor to_device<bfloat8_b>(
     return to_device<uint32_t>(tensor, target_device, memory_config, cq_id, sub_device_ids);
 }
 
+// ======================================================================================
+//     Helpers for converting between logical <-> physical data with full tensor spec
+// ======================================================================================
+namespace CMAKE_UNIQUE_NAMESPACE {
```

Comment:
Actually - how does this work? Would this be equivalent to unnamed namespace in C++?

Reply:
@sminakov-tt Would probably know more about the details.